relayer/src/worker/packet.rs

```diff
@@ -172,6 +175,7 @@ fn handle_packet_cmd<ChainA: ChainHandle, ChainB: ChainHandle>(
                 clear_interval,
                 height,
             ) {
+                scheduled_packet_clearing = true;
```

Comment:
Could we maybe instead remove the mutation of `is_first_run` from `should_clear_packets` (with this name it should never do that ;)) and instead only set it to `false` if `schedule_packet_clearing` is Ok below?